piqueserver/player.py

```diff
@@ -403,16 +403,7 @@ def on_hack_attempt(self, reason):
         self.kick(reason)
 
     def on_user_login(self, user_type, verbose=True):
-        if user_type == 'admin':
-            self.admin = True
-            self.speedhack_detect = False
```

Comment:
Disabling speedhack detection is something that should probably be kept.

Reply:
This has moved to `set_user_type` method.